packages/datadog-plugin-next/test/index.spec.js

```diff
@@ -71,11 +72,16 @@ describe('Plugin', function () {
         })
       })
 
-      after(() => {
-        execSync(`rm ${__dirname}/package.json`)
-        execSync(`rm ${__dirname}/package-lock.json`)
-        execSync(`rm -rf ${__dirname}/node_modules`)
-        execSync(`rm -rf ${__dirname}/.next`)
+      after(function () {
+        this.timeout(5000)
```

Comment:
do you need to set the timeout here too? I think if you call this on the `before` it affects the whole `describe`

Reply:
This after step sometimes times out so I bumped the limit slightly to avoid flakes.